bin/rspecq

```diff
@@ -91,6 +97,7 @@ opts[:file_split_threshold] ||= Integer(ENV["RSPECQ_FILE_SPLIT_THRESHOLD"] || 99
 opts[:report] ||= env_set?("RSPECQ_REPORT")
 opts[:report_timeout] ||= Integer(ENV["RSPECQ_REPORT_TIMEOUT"] || DEFAULT_REPORT_TIMEOUT)
 opts[:max_requeues] ||= Integer(ENV["RSPECQ_MAX_REQUEUES"] || DEFAULT_MAX_REQUEUES)
+opts[:fail_fast] ||= Integer(ENV["RSPECQ_FAIL_FAST"]) rescue nil
```

Comment:
We should only set this if `ENV["RSPECQ_FAIL_FAST"]` is set.

Reply:
Integer(x) will fail if it is not set, thus defaulting to nil (e.g. fail fast disabled)